src/App.js

```diff
@@ -0,0 +1,30 @@
+import HomePage from './components/HomePage';
+import MoviePage from './components/MoviePage';
+import ErrorBoundary from './components/ErrorBoundary';
+import { BrowserRouter as Router, Route, Switch} from 'react-router-dom';
+import SearchPage from './components/SearchPage';
+
+function App() {
+  return (
+    <Router>
+      <ErrorBoundary>
+        <Switch>
+          <Route exact path="/">
+            <HomePage/>
+          </Route>
+          <Route path="/film/:id">
+            <MoviePage />
+          </Route>
+          <Route path="/search/:movie/:searchBy">
```

Comment:
This path will already be covered from below Route path - "/search/:movie" as there is no exact prop passed